protos/feast/core/CoreService.proto

```diff
@@ -94,3 +98,11 @@ message GetFeastCoreVersionRequest {}
 message GetFeastCoreVersionResponse {
     string version = 1;
 }
+
+message UpdateStoreRequest {
+    feast.core.Store store = 1;
+}
+
+message UpdateStoreResponse {
```

Comment:
Are we going to use gRPC errors to notify the caller of problems in the request?